src/common/focusKeeper/__tests__/FocusKeeper.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import FocusKeeper from '../FocusKeeper';
+
+// Note about tests:
+// @testing-library fireEvent("keyPress",tab) does not simulate tab correctly
+// @testing-library/user-event provides an alternative: userEvent.tab(),
+// but it does not simulate focus redirect correctly. FocusKeeper's focus is not redirected to target
+// That's why ended up just using element.focus() for testing
+
+describe('<focusTrapper /> ', () => {
+  it('keeps focus inside', async () => {
+    let focusEventCount = 0;
+    const focusListener = (e: React.FocusEvent<HTMLInputElement>) => {
+      const id = e.target.getAttribute('id');
+      if (id === 'target') {
+        focusEventCount = focusEventCount + 1;
+      }
+    };
```

Comment:
Second listener was obsolete, so removed it. Kept same function name, because it is a focusListener (now for single element), but changed focusEventCount to focusEventsOnTarget